### PR TITLE
Updated contact us system

### PR DESCRIPTION
**Pull request by samuelgrant**
** Dump Cloudflare Cache **

~~ WIP ~~

This commit changes the contact us UI(s) - Change Log:
- Replaces the three Blade contact pages with one React/Blade hybrid page
- Updated the Navbar, Email and Sitemap URLs
- Deleted old Blade files
- Overhauled the admin 'read mail' page
- Introduced basic React components for the project including a spinner, form controls and a helper file
- Tidied routes

** TODO: **
- [x] User feedback
- [ ] Improve the HTML email template - Make it look like the admin page for viewing emails
- [ ] (?) Allow enable/disable of certian contact types
- [x] Fix the validator
- [x] Fix the reCAPTCHA

**Link to developer docs:**

#### Will this PR create any issues?

### CODE REVIEW

File: resources/views/contact_us.blade.php [hunk 1]
```diff
@@ -0,0 +1,9 @@
+@extends('layouts.app')
+
+@section('content')
+	<script src="https://www.google.com/recaptcha/api.js" async></script>
+	@include('layouts.messages')
+  	<div class="container conform">	
+    	<div id="ranchor_contactUs" class="mt-5 mb-5" style="padding: 20px 40px;"></div>
```
Comment (samuelgrant) [inline resources/views/contact_us.blade.php]: indent by one tab

File: resources/views/contact_us.blade.php [hunk 2]
```diff
@@ -0,0 +1,9 @@
+@extends('layouts.app')
+
+@section('content')
+	<script src="https://www.google.com/recaptcha/api.js" async></script>
```
Comment (samuelgrant) [inline resources/views/contact_us.blade.php]: I want to try removing this, it's already in the head.

File: app/Mail/Enquiry.php [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+     * Build the message.
+     *
+     * @return $this
+     */
+    public function build()
+    {
+        $this->replyTo($this->email);
+        return $this->view('emails.contact.enquiryReceived')// needs to be fixed
+                    ->text('emails.contact.enquiryReceivedPlain')// needs to be fixed
+                    ->subject('New '.ucfirst($this->enquiryType).' Enquiry Received');
```
[the remark this answers is not in the document]
Comment (MitchellQ) [reply app/Mail/Enquiry.php]: removed

File: resources/assets/js/components/FormControl.js [hunk 4]
```diff
@@ -0,0 +1,285 @@
...
+        if (!this.props.type)
+            throw 'You must provide a type for the input field';
+
+        // Check against our allowed types
+        if (Type.includes(this.props.type))
+            return this.props.type;
+
+        // No type found? - use TEXT as default
+        
+        return 'text';
```
[the remark this answers is not in the document]
Comment (MitchellQ) [reply resources/assets/js/components/FormControl.js]: Removed

File: resources/assets/js/pages/ContactUs.Partials.js [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+    render() {
+        const { message } = this.props.fields;
+
+        return (
+            <FormGroup className="form-group col col-12" label='Your Message:' htmlFor='message' required>
+                <TextArea id='message' maxLength='1000' rows='6' required value={message}
+                    onChange={this.props.onChange.bind(this, 'fields', 'message')} 
+                />
+            </FormGroup>
+        )//this.props.onChange.bind(this, 'fields', 'message')
```
[the remark this answers is not in the document]
Comment (MitchellQ) [reply resources/assets/js/pages/ContactUs.Partials.js]: Deleted

File: resources/assets/js/pages/ContactUs.js [hunk 6]
```diff
@@ -0,0 +1,162 @@
...
+                message: '',
+                company: '',
+                sponsorshipTypes: [''],
+                subscribe: true
+            }
+        }
+    }
+
+    componentDidMount(){
+        this._getData();//If Any
```
[the remark this answers is not in the document]
Comment (MitchellQ) [reply resources/assets/js/pages/ContactUs.js]: Removed

File: resources/assets/js/pages/ContactUs.js [hunk 7]
```diff
@@ -0,0 +1,162 @@
...
+    }
+
+    _getData(){
+        // ajax call to get data???
+        // Maybe block some contact types during the year
```
[the remark this answers is not in the document]
Comment (MitchellQ) [reply resources/assets/js/pages/ContactUs.js]: Removed

File: resources/views/contact_us.blade.php (hunk 1 above)
Comment (samuelgrant) [reply resources/views/contact_us.blade.php]: done

**Pull request merged**